config/_default/menus/menus.en.yaml

```diff
@@ -1453,61 +1453,146 @@ main:
     weight: 106
   - name: Java
-    url: tracing/trace_collection/open_standards/java/
+    url: tracing/trace_collection/otel_instrumentation/java/
+    parent: tracing_otel_inst
+    identifier: tracing_otel_inst_java
+    weight: 101
+  - name: Python
+    url: tracing/trace_collection/otel_instrumentation/python/
+    parent: tracing_otel_inst
+    identifier: tracing_otel_inst_python
+    weight: 102
+  - name: Ruby
+    url: tracing/trace_collection/otel_instrumentation/ruby/
+    parent: tracing_otel_inst
+    identifier: tracing_otel_inst_ruby
+    weight: 103
+  - name: Go
+    url: tracing/trace_collection/otel_instrumentation/go/
+    parent: tracing_otel_inst
+    identifier: tracing_otel_inst_go
+    weight: 104
+  - name: NodeJS
+    url: tracing/trace_collection/otel_instrumentation/nodejs/
+    parent: tracing_otel_inst
+    identifier: tracing_otel_inst_nodejs
+    weight: 105
+  - name: C++
+    url: tracing/trace_collection/otel_instrumentation/cpp/
+    parent: tracing_otel_inst
+    identifier: tracing_otel_inst_cpp
+    weight: 107
+  - name: .NET
+    url: tracing/trace_collection/otel_instrumentation/dotnet/
+    parent: tracing_otel_inst
+    identifier: tracing_otel_inst_dotnet
+    weight: 108
+  - name: Instrumenting with OpenTracing
+    url: tracing/trace_collection/opentracing/
+    parent: tracing_trace_collection
+    identifier: tracing_open_standards
+    weight: 107
+  - name: Java
+    url: tracing/trace_collection/opentracing/java/
     parent: tracing_open_standards
     identifier: tracing_open_standards_java
     weight: 101
   - name: Python
-    url: tracing/trace_collection/open_standards/python/
+    url: tracing/trace_collection/opentracing/python/
     parent: tracing_open_standards
     identifier: tracing_open_standards_python
     weight: 102
   - name: Ruby
-    url: tracing/trace_collection/open_standards/ruby/
+    url: tracing/trace_collection/opentracing/ruby/
     parent: tracing_open_standards
     identifier: tracing_open_standards_ruby
     weight: 108
   - name: Go
-    url: tracing/trace_collection/open_standards/go/
+    url: tracing/trace_collection/opentracing/go/
     parent: tracing_open_standards
     identifier: tracing_open_standards_go
     weight: 104
   - name: NodeJS
-    url: tracing/trace_collection/open_standards/nodejs/
+    url: tracing/trace_collection/opentracing/nodejs/
     parent: tracing_open_standards
     identifier: tracing_open_standards_nodejs
     weight: 105
   - name: PHP
-    url: tracing/trace_collection/open_standards/php/
+    url: tracing/trace_collection/opentracing/php/
     parent: tracing_open_standards
     identifier: tracing_open_standards_php
     weight: 106
   - name: .NET
-    url: tracing/trace_collection/open_standards/dotnet/
+    url: tracing/trace_collection/opentracing/dotnet/
     parent: tracing_open_standards
     identifier: tracing_open_standards_dotnet
     weight: 108
+  - name: Propagating Trace Context
```

Comment:
Because that info is the same for DD and Otel, I'm going to pull it out into its own page (it'll have the the language buttons at the top, making it look like one page even though here in the nav it has to be listed as 7 different pages). Readers can find it confusing when we dump them into a page of similar-but-slightly-different content for a little piece of common content. Both the DD and the OTel sets of pages will have links to the propagation info